plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -744,6 +785,15 @@ fu_plugin_update_online (FuPlugin *plugin,
 		return FALSE;
 	}
 
+	if (!fu_plugin_thunderbolt_validate_firmware (udevice, blob_fw, &error_local)) {
+		g_set_error (error,
+			     FWUPD_ERROR,
+			     FWUPD_ERROR_NOT_FOUND,
```

Comment:
`FWUPD_ERROR_INVALID_FILE` maybe?

Reply:
Sure. My bad. I used it in other places and forgot to change this one.

BTW, I see I wasn't consistent about adding a space after !
What is the project style, add it or not?

Reply:
No space after the `!`